marketplace-ui/src/app/modules/product/product.component.html

```diff
@@ -36,8 +36,17 @@ <h4 class="text-secondary">
       }
     </div>
   } @else {
-    <div class="h1 product-container text-center align-items-center">
-      {{ 'common.nothingFound' | translate }}
+    <div
+      class="d-flex flex-column align-items-center justify-content-center w-100">
+      <div class="not-found-result w-100">
+        <img
+          src="/assets/images/misc/search-off.svg"
+          alt="Search not found"
+          class="no-result-image mx-auto d-block" />
+        <div
+          class="h5 text-center not-found-text"
```

Comment:
![image](https://github.com/user-attachments/assets/5b202e9f-3778-4891-9d26-8980f6ed1162)
the text have suspicious margin